README.md

```diff
@@ -111,38 +94,38 @@ A table containing any of the following keys may be passed into the Conctr const
 
 | Key | Data type | Default value | Description |
 | ----| --------------- | --------- | ----------- |
-| isEnabled | Boolean | `true` | When enabled, location data will be automatically included with the data payload|
-| interval | Integer | 3600000|  Duration in milliseconds since last location update to wait before sending a new location |
-| sendOnce | Boolean | `false` | Setting to `true` sends the location of the device only once when the device restarts |
+| send_loc | Boolean | `true` | When enabled, location data will be automatically included with the data payload|
```

Comment:
Not sure why you switched from camel case. Put it back here and in the readme.
send_loc -> sendLoc, etc.